### ISSUE TITLE
Support a UI picker of all available targets

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode-chrome-debug-core#19]
See Microsoft/vscode#6569

**Comment by roblourens** [comment]
Not that useful without https://github.com/nodejs/node/issues/8464, so I'm going to postpone it. Also see https://bugs.chromium.org/p/chromium/issues/detail?id=649823#

**Comment by auchenberg** [comment]
We should implement this feature for existing running processes, so `url` param can be omitted an  UI selector will be prompted instead.

Once Node/Electron/Chrome lands the ability to enable debugging on-the-fly we can expand the usefulness of this feature.

**Comment by auchenberg** [comment]
https://github.com/Microsoft/vscode-chrome-debug/issues/280

**Comment by auchenberg** [comment]
@roblourens I think this has landed in Node master, as the old V8 protocol has been deprecated. It would be good to get a prototype going.

I imagine a flow like:

1) List all target processes in a drop down component (node/chrome/edge/etc)
2) On selection SIGUSR1 is sent to the process
3) A request to http/json is made
4) If more than one relevant target is returned, show new drop down with the targets
5) Selected or first target is used to connect the debugger.

This would made the attach scenario much simpler for Chrome, and enable a good experience for Node

**Comment by roblourens** [comment]
Cool, I'll check it out. I'll have to check the version of node that's running before displaying it, hopefully that won't lead to a confusing experience.

**Comment by roblourens** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=649823# was closed and Chrome will not support dynamically entering debug mode anytime soon. I'll continue to use Microsoft/vscode-chrome-debug#280 to track the chrome-specific task of showing a picker of tabs, and https://github.com/Microsoft/vscode/issues/25036 to track the node-specific task of entering debug mode dynamically and showing a list of processes.
